js/markupBlog.js

```diff
@@ -152,41 +246,69 @@ const markupMain = () => {
     const commentIcon = createElem({
       nodeType: "img",
       attribute: [
-        { name: "src", value: elem.icon },
+        { name: "src", value: "../images/atoms/a-icon-text.svg" },
         { name: "alt", value: "icon" },
       ],
     });
-    const author = markupAuthor(elem);
+    const author = createElem({
+      nodeType: "div",
+      className: "author",
+    });
+    const authorInfo = createElem({
+      nodeType: "div",
+      className: "author_info",
+    });
+    const authorImg = createElem({
+      nodeType: "img",
+      className: "author_img",
+      attribute: [
+        {
+          name: "src",
+          value: "../images/authors/Grace.png",
+        },
+        { name: "alt", value: "author photo" },
+      ],
+    });
+    const authorName = createElem({
+      nodeType: "p",
+      className: "author_name",
+      text: elem.author,
+    });
+    const comment = markupDescription();
+
+    data.blog.content.textBlog[0].author.description.stars.forEach((item) => {
```

Comment:
you shouldn't use mock json data, instead you will use data from the service call, please change all mock to real data